Dockerfile.compute-node

```
@@ -754,16 +773,27 @@ RUN rm /usr/local/pgsql/lib/lib*.a
 # Extenstion only
 #
 #########################################################################################
+FROM python:3.11-slim-bookworm AS generate-ext-index
```

Let's align with the rest of the images we use

```suggestion
FROM python:3.9-slim-bullseye AS generate-ext-index
```